clients/client-python/gravitino/client/gravitino_metalake.py

```diff
@@ -56,40 +54,32 @@ def __init__(self, metalake: MetalakeDTO = None, client: HTTPClient = None):
         )
         self.rest_client = client
 
-    def list_catalogs(self, namespace: Namespace) -> List[NameIdentifier]:
-        """List all the catalogs under this metalake with specified namespace.
-
-        Args:
-            namespace The namespace to list the catalogs under it.
+    def list_catalogs(self) -> List[str]:
+        """List all the catalogs under this metalake.
 
         Raises:
             NoSuchMetalakeException if the metalake with specified namespace does not exist.
 
         Returns:
-            A list of {@link NameIdentifier} of the catalogs under the specified namespace.
+            A list of the catalog names under this metalake.
         """
-        Namespace.check_catalog(namespace)
-        url = f"api/metalakes/{namespace.level(0)}/catalogs"
+        url = f"api/metalakes/{self.name()}/catalogs"
         response = self.rest_client.get(url)
         entity_list = EntityListResponse.from_json(response.body, infer_missing=True)
         entity_list.validate()
-        return entity_list.identifiers()
+        return [identifier.name for identifier in entity_list.identifiers()]
```

Comment:
`name` is a method in `NameIdentifier`, so it should `identifier.name()` here

Reply:
solved! and add a test case "test_catalogy.py" to cover it. please review again, thanks!